plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/glue/GlueHiveMetastoreConfig.java

```diff
@@ -203,6 +204,19 @@ public GlueHiveMetastoreConfig setAwsCredentialsProvider(String awsCredentialsPr
         return this;
     }
 
+    public Optional<String> getAwsCredentialsProviderConf()
```

Comment:
If we are pointing it to a particular file then we could add some annotations like `@FileExists`

Reply:
Thanks for the review.